doc/thirdpartypackages/index.rst

```diff
@@ -284,3 +284,16 @@ existing table. See the `blume github repository
 
 .. image:: /_static/blume_table_example.png
 
+GUI tool kit
```

Comment:
```suggestion
GUI applications
```
In the Matplotlib terminology, [toolkits](https://matplotlib.org/api/toolkits/index.html)  are add-ons on the library level.